charts/chronicle/templates/opa-init.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.opa.enabled }}
+{{$stlServiceName := include "lib.call-nested" (list . "sawtooth" "common.names.fullname")}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-opa-init
+  labels: {{ include "chronicle.labels" . | nindent 4 }}
+    component: opa-init
+spec:
+  backoffLimit: 5
+  template:
+    metadata:
+      labels: {{- include "chronicle.labels" . | nindent 8 }}
+        component: opa-init
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ include "lib.serviceAccountName" . }}
+      automountServiceAccountToken: true
+      affinity: {{ include "lib.safeToYaml" .Values.affinity | nindent 8 }}
+      containers:
+        - name: opa-init
+          {{- include "lib.image" (dict "imageRoot" .Values.opa.opaInit.image "global" .Values.global ) | nindent 10 }}
+          command: [ "bash", "-c"]
+          args:
+            - -exc
+            - |
+              echo "Waiting 20 seconds for Sawtooth to start";
```

Comment:
`wait-for-it` should be in this container.  If not add it, and use it.  Sleeps are not good enough

Reply:
Again, got it in a ConfigMap for now. Could we publish a `wait-for-it` container for this purpose?